8.x/getting-started/documentation.md

```diff
@@ -2,4 +2,466 @@
 
 [[toc]]
 
-Larammerce is getting documented, as well as developed at high speed. So if you want to be on track, follow the guides.
\ No newline at end of file
+This section guides you to write a document for larammerce project .
+
+## Overall Structure
+
+Each larammerce document should have the parts below :
+
+**1. Introduction :** Describe the overall view of the page , e.g. the subject it is talking about .
```

Comment:
a principle in writing is to not use space before a comma sign but after. (after page)
this sentence below is more formal : 
Describe the overall view of the page, e.g. write a general illustration of the represented subject.